account_analytic_project/model/account_analytic_account.py

```diff
@@ -0,0 +1,77 @@
+# -*- encoding: utf-8 -*-
+"""Extend account.analytic.account."""
```

Comment:
@NL66278 You mean MQT now requires a module-level docstrings? Strange to me. But I did not follow recent MQT developments.